runtime/shared_common/OSCachemmap.cpp

```diff
@@ -1419,7 +1419,7 @@ SH_OSCachemmap::getCacheStats(J9JavaVM* vm, const char* ctrlDirName, UDATA group
 	SH_OSCachemmap *cache = NULL;
 	void *cacheHeader;
 	I_64 *timeValue;
-	I_32 inUse;
+	UDATA inUse = (UDATA)J9SH_OSCACHE_UNKNOWN;
```

Comment:
The casts to `UDATA` here and elsewhere in this file will not be needed if the definition of `J9SH_OSCACHE_UNKNOWN` is changed as suggested.

Reply:
I've made the change but added a cast to `(I_64)` in some places.

Reply:
A direct cast from `UDATA` to `I_64` is not appropriate: on 32-bit platforms that involves both a signed-ness change and a size change.

Reply:
The cast on line 1422 is now redundant.